cbmc/proofs/IotMqtt_Init/IotMqtt_Init_harness.c

```diff
@@ -0,0 +1,10 @@
+#include "iot_config.h"
+#include "private/iot_mqtt_internal.h"
+
+#include <stdlib.h>
+#include <string.h>
+
+void harness()
+{
+  IotMqtt_Init();
```

Comment:
I assume this initializes a global structure? Did we make sure to run with `--nondet-static` or whatever the flag is?

Reply:
Yes, take a look at `cbmc-batch.yaml `, it adds the `--nondet-static` flag.